nomad/riscv64.patch

```diff
@@ -0,0 +1,42 @@
+diff --git PKGBUILD PKGBUILD
+index 3f2156c6..c66c57d5 100644
+--- PKGBUILD
++++ PKGBUILD
+@@ -25,15 +25,18 @@ source=(
+   'systemd.service'
+   'tmpfiles.conf'
+   'defaults.hcl'
++  'add_riscv_support.patch'
+ )
+ sha512sums=('22a741d83f6d0724e8bfb0fd26f8a5f34e7e8a0e7d340b313c4a8c8abfe9ab627064864822bd544f27e8bfab68967aa20f5ab73e6ca27b29cc7091a8d1f39d18'
+             '4fe233272e6872cb9ec0c594944a3c701d93ee59952688a4bc45b126442ecf973d23e50b3c4c9c5c62778d0c555ace2e2781955448239249d39fe21d184c8f1b'
+             '398724820ee6f1ffa7860b8dd0013b671ba97ffb001331a10e7795148e3e6ff711a06f2156187c23b71bfccb13dac70828df5a29d59441706753cc17afd1e3d7'
+-            '9bc9dc93b2683c848e94d5d7a59102ceebd6c4b64f14c85591b9502bc44e3c74589425f1f278ef21407d10c4a9a82abd95e2c748d347d76836bdb741bbff7f7c')
++            '9bc9dc93b2683c848e94d5d7a59102ceebd6c4b64f14c85591b9502bc44e3c74589425f1f278ef21407d10c4a9a82abd95e2c748d347d76836bdb741bbff7f7c'
++            'ca488369c4b3ef0dddf66bb9a72847b17b7ff40087f7ce6224db07a1721ecd52f394a589ee7635410846a2b069e7a734f87fc8907543a4ff01f2fc82d42f72ef')
+ b2sums=('eb5ed863c14b5755a295452ee23c16722ab89a051c13ba18446fbd65162b533ff363797792f00be801f23a8a7380750e6701efccf72aef6c64fc9be2219b0aa1'
+         '0f931868e9d7465f3be176ab789b7c7b5a4c35713a82e4d3bb94460254011779d814d4708f4f8cc76ba0dd1928ac7eeda42e01ed43be675a5d26d0b75ba986b9'
+         '4142a758057382846b48e9bfc443c1218c8fde866544d56c6e773933d67a0824e365280d39864249f355aee6080e0b4b969fd8cc4c5826583821ad1843b07e35'
+-        'abbf4c69d70e03b318f73ef84ccf258e39b25ac956ec68c82bebe277662ddd1677719c3b693c04a8cc38554c562a127aa2ba4061e20e594440d432d729bab3b3')
++        'abbf4c69d70e03b318f73ef84ccf258e39b25ac956ec68c82bebe277662ddd1677719c3b693c04a8cc38554c562a127aa2ba4061e20e594440d432d729bab3b3'
++        'b82609e4d183d8b82f693a84be388b150c6669326c4d4e43174da4b2f702e4686ae46a13f5a22444ae961dcb5049c31386223df0163997d628b2924d6dff9cc5')
+ 
+ prepare() {
+ 	cd "$pkgname-$pkgver"
+@@ -43,6 +46,8 @@ prepare() {
+ 
+   # download dependencies
+   go mod download
++  go mod vendor
++  patch -Np1 -i ../add_riscv_support.patch
```

Comment:
I suggest change the vendor to an active fork, and send the vendor change to upstream
```suggestion
+  go mod edit -replace github.com/kr/pty=github.com/creack/pty
+  go mod edit -replace github.com/boltdb/bolt=github.com/etcd-io/bbolt
   go mod download
```